lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
 
-    ThrowingCallable createClassificationCall =
-        () -> {
-          CLASSIFICATION_SERVICE.createClassification(classification);
+          classification = classificationService.createClassification(classification);
+
+          assertThat(classification.getServiceLevel()).isEqualTo("P0D");
         };
 
-    assertThatThrownBy(createClassificationCall).isInstanceOf(NotAuthorizedException.class);
+    return DynamicTest.stream(iterator, c -> String.format("for '%s'", c), test);
   }
 
-  @WithAccessId(user = "admin")
+  @WithAccessId(user = "businessadmin")
   @Test
-  void should_SetDefaultServiceLevel_When_TryingToCreateClassificationWithMissingServiceLevel()
-      throws Exception {
-    Classification classification =
-        CLASSIFICATION_SERVICE.newClassification("newKey718", "", "TASK");
-    classification = CLASSIFICATION_SERVICE.createClassification(classification);
-    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+  void should_SetDefaultValues_When_CreatingClassificationWithoutSpecificValues() throws Exception {
+    Classification classification = classificationService.newClassification("Key7", "", "TASK");
+    classification = classificationService.createClassification(classification);
 
-    classification = CLASSIFICATION_SERVICE.newClassification("newKey71", "", "TASK");
-    classification.setServiceLevel("");
-    classification = CLASSIFICATION_SERVICE.createClassification(classification);
     assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+    assertThat(classification.getId()).isNotEqualTo(null);
+    assertThat(classification.getId()).isNotEmpty();
+    assertThat(classification.getCreated()).isNotNull();
+    assertThat(classification.getModified()).isNotNull();
+    assertThat(classification.getParentId()).isEqualTo("");
+    assertThat(classification.getParentKey()).isEqualTo("");
+    assertThat(classification.getIsValidInDomain()).isEqualTo(false);
   }
 
-  @WithAccessId(user = "admin")
-  @Test
-  void should_SetDefaultServiceLevel_When_TryingToUpdateClassificationWithMissingServiceLevel()
-      throws Exception {
-    Classification classification =
-        CLASSIFICATION_SERVICE.getClassification("CLI:000000000000000000000000000000000001");
-    classification.setServiceLevel(null);
-    classification = CLASSIFICATION_SERVICE.updateClassification(classification);
-    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+  @WithAccessId(user = "businessadmin")
+  @AfterAll
+  void should_NotCreateClassification_When_ExceptionIsThrownDuringCreation() {
+    List<ClassificationSummary> classifications =
+        classificationService.createClassificationQuery().keyIn("KeyErrCreation").list();
 
-    classification.setServiceLevel("");
-    classification = CLASSIFICATION_SERVICE.updateClassification(classification);
-    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+    assertThat(classifications).isEmpty();
+  }
```

Comment:
Not necessary. @gitgoodjhe When an exception is thrown we assume that the given entity is not created. Is that correct?

Reply:
Correct...Exception = program disrupted = no db entry